Cannot find module 'slimerjs'

Using node v7.2.1 (npm v3.10.10)

**install slimerjs locally**
/tmp/test $ npm i slimerjs@*
test@1.0.0 /tmp/test
└── slimerjs@0.10.2 

**install slimerjs globally also**
/tmp/test $ npm i -g slimerjs@*
/home/bob/.nvm/versions/node/v7.2.1/bin/slimerjs -> /home/bob/.nvm/versions/node/v7.2.1/lib/node_modules/slimerjs/src/slimerjs-node
/home/bob/.nvm/versions/node/v7.2.1/lib
└── slimerjs@0.10.2 

$ which node
/home/bob/.nvm/versions/node/v7.2.1/bin/node

**/tmp/test/test.js contains**
```
console.log(require('phantomjs').path);
console.log(require('slimerjs').path);
```
/tmp/test $ node test.js 
/tmp/test/node_modules/phantomjs/lib/phantom/bin/phantomjs
module.js:472
    throw err;
    ^

**Error: Cannot find module 'slimerjs'**
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/tmp/test/test.js:9:13)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)

- Firefox: Mozilla Firefox 50.0.2
- Operating system: Ubuntu 14.04

--- a/package.json
+++ b/package.json
@@ -5,6 +5,7 @@
   "bin": {
     "slimerjs": "./src/slimerjs-node"
   },
+  "main": "./src/phantom-protocol.js",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/laurentj/slimerjs.git"

--- a/src/phantom-protocol.js
+++ b/src/phantom-protocol.js
@@ -1,0 +1,5 @@
+/**
+ * Exposes necessary information so slimerjs will work with phantom drivers.
+ */
+var path = require('path');
+exports.path = path.join(__dirname, '../', 'src', 'slimerjs');

